modules/end-user-guide/partials/proc_creating-a-workspace-from-workspace-sample.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// creating-a-workspace-from-included-options
+
+[id="creating-a-workspace-from-workspace-sample_{context}"]
+= Creating a workspace from workspace sample
```

Comment:
from "a" workspace sample. We are not to omit articles to shorten things. *shrug*

Reply:
Yep, I just fixed this now, but the docs still have more of omitted articles in the headings.